components/brave_rewards/resources/android_page/brave_rewards_page.tsx

```diff
@@ -65,6 +65,10 @@ window.cr.define('brave_rewards', function () {
     getActions().onPromotions(properties)
   }
 
+  function claimPromotion (properties: Rewards.Captcha) {
```

Comment:
not needed as android don't have captcha

Reply:
Removed.